jwst/datamodels/schemas/referencefile.schema.yaml

```diff
@@ -1,32 +1,73 @@
-  properties:
-    meta:
-      type: object
-      properties:
-        reffile:
-          title: Information about the reference file
-          type: object
-          properties:
-            type:
-              title: Reference file type
-              type: string
-              fits_keyword: REFTYPE
-            pedigree:
-              title: The pedigree of the reference file
-              type: string
-              fits_keyword: PEDIGREE
-            description:
-              title: Description of the reference file
-              type: string
-              fits_keyword: DESCRIP
-            author:
-              title: Author of the reference file
-              type: string
-              fits_keyword: AUTHOR
-            useafter:
-              title: Use after date of the reference file
-              type: string
-              fits_keyword: USEAFTER
+type: object
+properties:
+  meta:
+    type: object
+    properties:
+      date:
+        title: Date this file was created (UTC)
+        type: string
+        fits_keyword: DATE
+      origin:
+        title: Organization responsible for creating file
+        type: string
+        fits_keyword: ORIGIN
+      time_sys:
+        title: principal time system for time-related keywords
+        type: string
+        fits_keyword: TIMESYS
+      filename:
+        title: Name of the file
+        type: string
+        fits_keyword: FILENAME
+      filetype:
+        title: Type of data in the file
+        type: string
+        fits_keyword: FILETYPE
+      model_type:
+        title: Type of data model
+        type: string
+        fits_keyword: DATAMODL
+      telescope:
+        title: Telescope used to acquire the data
+        type: string
+        default: JWST
+        fits_keyword: TELESCOP
+      reftype:
+        title: Reference file type
+        type: string
+        fits_keyword: REFTYPE
+      pedigree:
+        title: The pedigree of the reference file
+        type: string
+        fits_keyword: PEDIGREE
+      description:
+        title: Description of the reference file
+        type: string
+        fits_keyword: DESCRIP
+      author:
+        title: Author of the reference file
+        type: string
+        fits_keyword: AUTHOR
+      useafter:
+        title: Use after date of the reference file
+        type: string
+        fits_keyword: USEAFTER
+      instrument:
+        title: Instrument configuration information
+        type: object
+        properties:
+          name:
+            title: Instrument used to acquire the data
+            type: string
+            enum: [NIRCAM, NIRSPEC, MIRI, TFI, FGS, NIRISS, ANY, N/A]
+            fits_keyword: INSTRUME
+          detector:
+            title: Name of detector used to acquire the data
+            type: string
+            enum: [NRCA1, NRCA2, NRCA3, NRCA4, NRCALONG, NRCB1, NRCB2, NRCB3, NRCB4,
+              NRCBLONG, NRS1, NRS2, ANY, MIRIMAGE, MIRIFULONG, MIRIFUSHORT, NIRISS,
```

Comment:
I vote for having "NIRISS" removed from the list and no longer treat it as deprecated. The few test data files laying around that might have this value will just need to be updated accordingly.